src/main/java/com/unzer/payment/paymenttypes/Applepay.java

```diff
@@ -49,11 +49,14 @@
     private String currencyCode;
     private BigDecimal transactionAmount;
 
-    public Applepay(String version, String data, String signature, ApplepayHeader applepayHeader) {
+    public Applepay() {
+    }
```

Comment:
What for?

Reply:
This is for the JSON-Library which we are using. This needs an empty public constructor to be able to map a JSON into this object.

Reply:
ok